src/cq_cam/command.py

```diff
@@ -79,12 +101,12 @@ def rel(cls, x=None, y=None, z=None, **kwargs):
         warnings.warn("Relative CV is deprecated", DeprecationWarning)
         return cls(end=RelativeCV(x=x, y=y, z=z), **kwargs)
 
-    def print_modal(self, previous: Optional[Command]):
+    def print_modal(self, previous: Optional[MotionCommand]):
         if self.modal and (previous is None or previous.modal != self.modal):
             return self.modal
         return ""
 
-    def xyz_gcode(self, start: cq.Vector, precision=3) -> (str, cq.Vector):
+    def xyz_gcode(self, start: cq.Vector, precision=3) -> Tuple[str, cq.Vector]:
```

Comment:
Good catch with a new catch:

I learned this quite recently myself too, but `Tuple[...]` is deprecated ( https://docs.python.org/3/library/typing.html#typing.Tuple ) since Python 3.9. The new official way to do it now is  `tuple[...]`  ([PEP 585](https://peps.python.org/pep-0585/))

Reply:
Good to know I will do this change as well!